instance/migrations/0048_appserver_refactor1.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-05-04 04:29
```

Comment:
@bradenmacdonald Do we want to keep this? I don't have a strong preference either way. But existing migration files don't have comments like this, so just making sure we make a conscious decision :)